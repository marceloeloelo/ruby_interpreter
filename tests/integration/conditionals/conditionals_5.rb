var = 1

if ( var && var )
  puts ("bien")
else
  puts ("mal")
end

if ( var || var )
  puts ("bien")
else
  puts ("mal")
end

if ( var && nil )
  puts ("mal")
else
  puts ("bien")
end

if ( var || nil )
  puts ("bien")
else
  puts ("mal")
end

if ( var && nil || nil && 1 )
  puts ("mal")
else
  puts ("bien")
end

if ( var && nil || nil && 1 )
  puts ("mal")
elsif ( true )
  puts ("bien")
end

if ( var >= 1 || var <= 1 )
  puts ("bien")
else
  puts ("mal")
end

if ( var > 5 && var >= 10 && var == nil || var )
  puts ("bien")
else
  puts ("mal")
end

