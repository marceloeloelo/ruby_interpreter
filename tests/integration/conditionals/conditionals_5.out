bien
bien
bien
bien
bien
bien
bien
bien
